packages/pilot/src/components/Property/style.css

```
@@ -22,3 +22,18 @@
     margin: calc(var(--spacing-small) - 5px) 0;
   }
 }
+
+@media print {
+
+  .value {
+
+    &:empty {
```

poderia juntar o seletor `.value:empty`, visto que não tem nenhuma regra definida apenas para o `.value`

```suggestion
    .value:empty {
```